packages/react-components/src/components/ErrorBar.test.tsx

```diff
@@ -3,19 +3,22 @@
 
```

Comment:
Changes in this file are unrelated to VideoGallery. Just wanted to avoid the unregistered icon warnings in the console when running the tests.